packages/app/src/app/overmind/namespaces/editor/actions.ts

```diff
@@ -70,7 +70,7 @@ export const sandboxChanged: AsyncAction<{ id: string }> = withLoadApp<{
     return;
   }
 
-  await effects.vscode.closeAllTabs();
+  // await effects.vscode.closeAllTabs();
```

Comment:
Why this close?

Reply:
Oh, it should actually be there. The reason it closes is because the tabs references models that references files of the previous Sandbox. 

I just needed to test something and it sneaked in